open_spiel/python/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -204,20 +213,26 @@ def _play_game(logger, game_num, game, bots, temperature, temperature_drop):
   logger.opt_print(" Starting game {} ".format(game_num).center(60, "-"))
   logger.opt_print("Initial state:\n{}".format(state))
   while not state.is_terminal():
-    root = bots[state.current_player()].mcts_search(state)
+    cur_bot = bots[state.current_player()]
     policy = np.zeros(game.num_distinct_actions())
-    for c in root.children:
-      policy[c.action] = c.explore_count
-    policy = policy ** (1 / temperature)
-    policy /= policy.sum()
-    if len(actions) >= temperature_drop:
-      action = root.best_child().action
-    else:
-      action = np.random.choice(len(policy), p=policy)
+    if type(cur_bot) == minimax.minimaxBot:
```

Comment:
prefer isinstance over type, though by the enum would be even better so it's all consistent.